api/src/main/java/net/wesjd/anvilgui/AnvilGUI.java

```diff
@@ -189,6 +189,17 @@ public void closeInventory() {
 		}
 	}
 
+	/**
+	 * Closes the inventory if it's open.
+	 * @param force whether it'll force the inventory to be fully closed
+	 */
+	public void closeInventory(boolean force) {
+		if (force) {
+			preventClose = false;
+		}
+		closeInventory();
```

Comment:
I think this `preventClose` should be return the old value if force is true like that:
```java
final boolean oldvalue = preventClose;
if (force) {
    preventClose = false;
}
closeInventory();
if (force) {
    preventClose = oldvalue;
}
```

Reply:
ok